### PR TITLE
Use simple open in copy engine

### PR DESCRIPTION
**Pull request by ayan-b**
Before raising the PR, here is a check list:

1) have you written unit tests for your code changes?
2) have you run "make format"?
3) are you requesting to "dev"?
4) have you updated the change log?
5) do you think that you can understand your changes after 6 month?
   5.1) can someone else understand your changes without your explanation?
6) are you pround of your code changes?
   6.1) do you have the feeling of achievement?
7) please add your name and github link to contributors.rst in alphabetical order.

### CODE REVIEW

File: moban/copy/__init__.py [hunk 1]
```diff
@@ -30,7 +29,7 @@ def get_template(self, template_file):
         real_file_name = utils.find_file_in_template_dirs(
             template_file, self.template_dirs
         )
-        with codecs.open(real_file_name, encoding="utf-8") as file_handle:
+        with open(real_file_name, "rb") as file_handle:
             return file_handle.read()
```
Comment (ayan-b) [inline moban/copy/__init__.py]: Referring https://github.com/moremoban/moban/issues/265#issuecomment-485063029:

> When saving, it should use 'wb' flag, treating the content as binary throughout the journey within moban.

We would write into the file here.

**Comment by chfw** [comment]
Utils.py does the file writing, in 'wb' mode.

**Comment by chfw** [comment]
Python 3 failed but python 2 tests passed.

**Comment by chfw** [comment]
def write_file_out(filename, content, strip=True, encode=True):

**Comment by ayan-b** [comment]
> Python 3 failed but python 2 tests passed.

Pretty strange, I will look into it. Seem something to do with encoding.

**Comment by chfw** [comment]
Do you have the file which triggered this PR? Can we have a test case using it and get it pass?

Please put it in a separate test file because it will serve us in the future as critical regression test suites.

**Comment by ayan-b** [comment]
This is the file: https://github.com/coala/artwork/blob/master/artwork/logo/coala_color.svg as found from https://gitlab.com/ayan-b/mobans/-/jobs/199585396.

**Comment by chfw** [comment]
I see

**Comment by chfw** [comment]
Please Write up a new test case to copy it and verify copy works ok. 

And this test case will not be updated when moban evolves because it will be the future that we do like to break.

**Comment by ayan-b** [comment]
@chfw, I have added the test in an existing file. Should I create a new file and add there?

**Comment by chfw** [comment]
A new file is better because it is to be kept as it is during refactoring. Other test cases that fail in refactoring can be updated but this one cannot be.

So if this test fails, it means we got a regression.

**Comment by ayan-b** [comment]
@chfw is the current structure ok?

**Pull request merged**